woocommerce/woocommerce
# Clean up tasklist progression headercard experiment

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #45471.

We have removed the experiment in https://github.com/woocommerce/woocommerce/pull/33426

This PR removed the rest unused the experiment code.

Original PR: https://github.com/woocommerce/woocommerce-admin/pull/8233



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Please check the original PR and confirm that the experiment code is removed.

1. Create a fresh site
2. Go to WooCommerce Home and confirm it shows task list with no errors




### Changelog entry







<details>

#### Significance




#### Type




#### Message 

Clean up tasklist progression headercard experiment

</details>

<details>

#### Comment 

</details>
